packages/@o3r/azure-tools/README.md

```diff
@@ -0,0 +1,42 @@
+<h1 align="center">Otter Azure tools</h1>
+<p align="center">
+  <img src="https://raw.githubusercontent.com/AmadeusITGroup/otter/main/.attachments/otter.png" alt="Super cute Otter!" width="40%"/>
+</p>
+
+## Description
+
+Various Azure DevOps tools
+
+## How to install
+
+This package can be used via `npx` command to executed on of the listed command line interfaces.
+
+```shell
+npx -p @o3r/azure-tools ...
+```
```

Comment:
not sure this part is very helpful
it also seems confusing that the `How to install` section explains how to not install